src/service/async_service_impl.cc

```diff
@@ -15,7 +15,7 @@ namespace authservice {
 namespace service {
 
 namespace {
-constexpr uint16_t kHealthCheckServerPort = 10004;
+constexpr uint16_t kHealthCheckServerPort = 9091;
```

Comment:
I'm not sure regarding this, seems like we had the same health check port (with the main one? I kept getting the port-in-use error).